tap_gitlab/__init__.py

```diff
@@ -22,6 +22,7 @@
     'start_date': None,
     'groups': '',
     'ultimate_license': False,
+    'fetch_bridges': False,
```

Comment:
I imagine this is off by default because it creates another API call for every pipeline like `jobs` does, and most projects aren't going to use bridge jobs (I had to look them up in the docs because I wasn't familiar with the feature). This makes sense to me.

A quirk of the way this is implemented here is that discovery will report that bridges are selected (even if `fetch_bridges=False`), but simply not output any.

If you want to match the behavior of `fetch_pipelines_extended` and `fetch_merge_request_commits`, you'll also want to also [cast it to a truthy value](https://github.com/staaam/tap-gitlab/blob/feat/support-bridges/tap_gitlab/__init__.py#L921) and [add it to `STREAM_CONFIG_SWITCHES`](https://github.com/staaam/tap-gitlab/blob/feat/support-bridges/tap_gitlab/__init__.py#L198). This will cause discovery to report bridges as unselected unless `fetch_bridges=True`.

That being said, we're moving away from using config options like `fetch_pipelines_extended` and `fetch_merge_request_commits` to preferring catalog for selection of columns, as it's more in-line with the Singer spec. These are still around for backwards compatibility.

A better way to handle this if we want `bridges` to be opt-in, to instead set `selected-by-default` to False for the stream. Something like:

```python
if resource == "bridges":
            mdata = metadata.to_list(metadata.write(metadata.to_map(mdata), (), 'selected-by-default', False))
```

We eventually want to do this for `fetch_pipelines_extended` and `fetch_merge_request_commits` too.

Unfortunately there's a [known bug in Meltano](https://gitlab.com/meltano/meltano/-/issues/2556) that will cause this to be selected by default anyway for Meltano users. See the full related discussion here: https://gitlab.com/meltano/tap-gitlab/-/merge_requests/34#note_501858073

With that in mind, I could see the case for either solution.